videozen-frontend/src/reducer/data-reducer.js

```diff
@@ -0,0 +1,144 @@
+import uuid from "react-uuid";
+
+const findPlaylistName = (allPlaylists, videoId) => {
+  // return allPlaylists.map((playlist) =>
+  //   playlist.videos.filter((video, playlist) =>
+  //     video.id === videoId ? playlist.name : ""
+  //   )
+  // );
```

Comment:
There are some commented out code in some files. If you don't intend to use them in the further iterations, you can remove them.